examples/enforcedefaults/app.py

```diff
@@ -1,12 +1,13 @@
-#!/usr/bin/env python3
+from pathlib import Path
 
 import connexion
 import jsonschema
-from connexion.decorators.validation import RequestBodyValidator
 from connexion.json_schema import Draft4RequestValidator
+from connexion.validators import JSONRequestBodyValidator
 
 
-def echo(data):
+# TODO: should work as sync endpoint when parameter decorator is fixed
```

Comment:
Currently the parameter decorator uses a sync decorator when the view function is sync, even when the framework is async. This doesn't work as the async body cannot be read by a sync decorator.